# old version does not have all features needed for checks on i686
diff --git a/crates/taplo/Cargo.toml b/crates/taplo/Cargo.toml
index 966ba1db30..7912352478 100644
--- a/crates/taplo/Cargo.toml
+++ b/crates/taplo/Cargo.toml
@@ -36,7 +36,7 @@ serde = { version = "1", features = ["derive"], optional = true }
 assert-json-diff = "2"
 criterion = "0.3"
 difference = "2.0.0"
-pprof = { version = "0.9.1", features = ["flamegraph", "criterion"] }
+pprof = { version = "0.13.0", features = ["flamegraph", "criterion"] }
 serde_json = "1"
 toml = "0.5"
 
